<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>神奇弹幕</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>房间</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>直播间房号</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="roomIdEdit">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>可以是签约短房号</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="showLiveDanmakuButton">
            <property name="text">
             <string> 显示实时弹幕</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>弹幕功能</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>弹幕获取间隔</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="refreshDanmakuIntervalSpin">
            <property name="suffix">
             <string>毫秒</string>
            </property>
            <property name="minimum">
             <number>500</number>
            </property>
            <property name="maximum">
             <number>60000</number>
            </property>
            <property name="singleStep">
             <number>500</number>
            </property>
            <property name="value">
             <number>2000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="refreshDanmakuCheck">
            <property name="text">
             <string> 启动</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>弹幕超时间隔</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="removeDanmakuIntervalSpin">
            <property name="suffix">
             <string>秒</string>
            </property>
            <property name="maximum">
             <number>3600</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="DiangeAutoCopyCheck">
            <property name="text">
             <string>点歌自动复制</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="languageAutoTranslateCheck">
            <property name="text">
             <string>外语自动翻译</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="AIReplyCheck">
            <property name="text">
             <string>AI智能回复</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="testDanmakuEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="testDanmakuButton">
            <property name="text">
             <string> 测试弹幕</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>定时任务</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="SetBrowserCookieButton">
            <property name="text">
             <string> 设置Cookie</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SetBrowserDataButton">
            <property name="text">
             <string>设置Data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SetBrowserHelpButton">
            <property name="text">
             <string>帮助</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="addTaskButton">
          <property name="text">
           <string> 添加定时任务</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="taskListWidget">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="SendMsgEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="SendMsgButton">
            <property name="text">
             <string>发送弹幕</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>自动发送</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QCheckBox" name="autoSendWelcomeCheck">
            <property name="text">
             <string>欢迎进入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="autoSendThankCheck">
            <property name="text">
             <string>感谢送礼</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSpinBox" name="sendCDSpin">
            <property name="suffix">
             <string>秒</string>
            </property>
            <property name="prefix">
             <string>冷却</string>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="autoWelcomeWordsEdit">
          <property name="plainText">
           <string>欢迎 %1 ~~~</string>
          </property>
          <property name="placeholderText">
           <string>欢迎语，多行随机发送一行</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="autoThankWordsEdit">
          <property name="plainText">
           <string>感谢 %1 赠送的 %2 ~~~</string>
          </property>
          <property name="placeholderText">
           <string>欢迎语，多行随机发送一行</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>347</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
